runtime/gc_realtime/RealtimeGC.hpp

```diff
@@ -106,9 +110,9 @@ class MM_RealtimeGC : public MM_GlobalCollector
 	 */
 private:
 protected:
-	virtual void internalPreCollect(MM_EnvironmentBase *env, MM_MemorySubSpace *subSpace, MM_AllocateDescription *allocDescription, U_32 gcCode);
-	virtual void internalPostCollect(MM_EnvironmentBase *env, MM_MemorySubSpace *subSpace);
-	virtual bool internalGarbageCollect(MM_EnvironmentBase *env, MM_MemorySubSpace *subSpace, MM_AllocateDescription *allocDescription);
+	void internalPreCollect(MM_EnvironmentBase *env, MM_MemorySubSpace *subSpace, MM_AllocateDescription *allocDescription, U_32 gcCode);
+	void internalPostCollect(MM_EnvironmentBase *env, MM_MemorySubSpace *subSpace);
+	bool internalGarbageCollect(MM_EnvironmentBase *env, MM_MemorySubSpace *subSpace, MM_AllocateDescription *allocDescription);
```

Comment:
These three functions should remain virtual... In general all methods declared virtual in `MM_Collector` should remain virtual